apps/sms/views/conversation/js/information.js

```diff
@@ -208,7 +208,17 @@ function createListWithMsgInfo(message) {
   var list = [];
   if (message.delivery === 'received' ||
       message.delivery === 'not-downloaded') { // received message
-    list.push(message.sender);
+    if (message.type === 'mms') {
+      var numbers = message.receivers;
+      if (message.receivers.indexOf(message.sender)==-1) {
+        numbers.push(message.sender);
+      }
+      numbers.forEach(function(number) {
+        list.push(number);
+      });
+    } else {
+      list.push(message.sender);
+    }
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator for more information about the awesome `...` operator.